src/functions.rs

```diff
@@ -207,14 +219,21 @@ pub fn is_executable(path: impl AsRef<Path>) -> io::Result<bool> {
 ///
 /// ```
 /// use permissions::access_syscall;
-/// use permissions::consts::{R_OK, W_OK, X_OK, F_OK};
+/// use permissions::consts::{R_OK, W_OK, F_OK};
+/// #[cfg(unix)]
+/// use permissions::consts::X_OK;
 ///
 /// fn main() -> std::io::Result<()> {
 ///     assert!(access_syscall("src/lib.rs", R_OK | W_OK)?);
+///     #[cfg(unix)]
 ///     assert!(access_syscall("/", R_OK | X_OK)?);
```

Comment:
To avoid having two `cfg(unix)`, you can move both to the end of the function, and wrap them in a block.

```rust
#[cfg(unix)]
{
    assert!(access_syscall("/", R_OK | X_OK)?);
    assert!(!access_syscall("src/lib.rs", X_OK)?);
}
```

Reply:
Done.